charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -47,8 +47,17 @@ spec:
       securityContext:
```

Comment:
**⚠️ ⚠️ ⚠️ ⚠️   Changes in `daemonset-windows.yaml are missing!!! ⚠️ ⚠️ ⚠️ ⚠️**

Reply:
Nice catch! Thanks :bow:

Reply:
Finally windows have partial support, just adding "none" persistence mode there.